### DISCUSSION TITLE
Icons change

### DISCUSSION THREAD

**Discussion by nivelsistemasbr** [rustdesk/rustdesk#2588] [General]
Dear friends
I'm compiling a custom version but I can't change the main window icon. It already replaces all the icon files of the project but the original icon continues to appear. Does anyone have an idea what I can do?

**Comment by Moeppelchen** [comment]
> Dear friends I'm compiling a custom version but I can't change the main window icon. It already replaces all the icon files of the project but the original icon continues to appear. Does anyone have an idea what I can do?

There are some base64 encoded icons that need to be replaced (: 
There are some neat tips and tricks explained here:
https://github.com/CW3MH6/rustdesk-customization#changing-the-application-icons

**Comment by nicolas-ecarnot-sitpi** [comment]
https://github.com/Sitpi/rustdesk/blob/master/flutter/assets/logo.svg
